views/json/resources/livesearch/friends.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+elgg_gatekeeper();
+
+$limit = get_input('limit', elgg_get_config('default_limit'));
+$query = get_input('term', get_input('q'));
+$input_name = get_input('name', 'members');
+
+$user = elgg_get_logged_in_user_entity();
+$dbprefix = elgg_get_config('dbprefix');
+
+elgg_set_http_header("Content-Type: application/json;charset=utf-8");
+
+echo elgg_list_entities([
+	'query' => $query,
+	'type' => 'user',
+	'limit' => $limit,
+	'sort' => 'name',
+	'order' => 'ASC',
+	'wheres' => [
```

Comment:
Move this to users with a friends_only check. This view just sets friends only to true to renders users